### DISCUSSION TITLE
Window: scrollTo smooth behavior Safari support

### DISCUSSION THREAD

**Discussion by albertopiva** [mdn/mdn-community#672] [Content] [answered]
Hello everybody,

This is my first time here, hoping to follow all the guidelines I would ask information about the smooth behavior of Window scrollTo API.
I noticed that the [documentation page](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo#browser_compatibility) specifies that the `options.behavior` is supported since Safari 14 but
> Safari does not have support for the smooth scroll behavior.
<br>

After some searches inside the Safari release documentation, I founded that from [Safari 15.4](https://developer.apple.com/documentation/safari-release-notes/safari-15_4-release-notes) they introduced the support of smooth scroll:

> Added support for the CSS scroll-behavior property and ScrollOptions, allowing smooth scrolling to anchors or via JavaScript.

Moreover, looking at the documentation of [WebKit 15.4](https://webkit.org/blog/12445/new-webkit-features-in-safari-15-4/) probably we can confirm the support of the smooth behavior via css but also Window.scrollTo().

> Developers can also control scroll behavior for an element with either the CSS scroll-behavior property or the behavior option in window.scroll(), window.scrollTo(), and window.scrollBy() methods in JavaScript. This new support gives developers the ability to choose between instantly jumping to a position in the viewport or smoothly animating the scroll operation.

Should we update the [documentation page](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo#browser_compatibility)?

Thank you,
Alberto

**Comment by caugner** [comment, marked as answer]
> Should we update the [documentation page](https://developer.mozilla.org/en-US/docs/MDN/Community/Communication_channels)?

Good catch, I believe you're referring to the browser compatiability table on the scrollTo pages for [Window](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo#browser_compatibility) and [Element](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollTo#browser_compatibility), which seem to be inconsistent with regard to Safari anyways.

So to your question: Definitely yes. Can you [open a data issue](https://github.com/mdn/browser-compat-data/issues/new/choose) in the browser-compat-data repo? 🙏

**Reply by albertopiva** [reply]
Yes sure, I don't know why the documentation page had a wrong link, I have updated it. In any case, only the [Window](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo#browser_compatibility) page needs the update.
